semcore/base-trigger/CHANGELOG.md

```diff
@@ -2,1079 +2,1088 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.49.0] - 2024-11-20
+
+### Fixed
+
+* Added `Hint` to the `FilterTrigger.ClearButton`.
+* Remove redundant `aria-label` & `aria-labelledby` from `FilterTrigger.ClearButton`.
+* Added `role` combobox to the `FilterTrigger.TriggerButton`.
+* Added `count` prop to `FilterTrigger.Counter`, along with screenreader only `selected` text
```

Comment:
let's also mention that placeholder text is now aria-hidden

Reply:
added